addon/ng2/blueprints/ng2/files/__path__/app/__name__.component.ts

```diff
@@ -3,8 +3,12 @@ import { Component } from '@angular/core';
 @Component({
   moduleId: module.id,
   selector: '<%= htmlComponentName %>-app',
-  templateUrl: '<%= htmlComponentName %>.component.html',
-  styleUrls: ['<%= dasherizedModuleName %>.component.<%= styleExt %>']
+  <% if(isMobile) { %>template: `
```

Comment:
seems like you're repeating `if (isMobile) ...`. Would that simplify the logic to put template and style next to each other in the if block?

Also, spaces between `if` and `(`.